docs/sources/reference/api/docker_remote_api_v1.16.md

```diff
@@ -1326,6 +1327,11 @@ Create a new image from a container's changes
              "Cmd":[
                      "date"
              ],
+	     "Labels": [
+                     "Vendor=Acme",
+                     "License=GPL",
+                     "Version=1.0"
+             ],
```

Comment:
Why are the labels stored/accessed as a dictionary/hash everywhere else, but provided as an array through the API?
What happens if we do something like:

```
"Labels": [
  "foo=bar",
  "foo=baz",
]
```

I think it would be more appropriate to treat it as a dictionary/hash everywhere.